src/services/code-index/processors/scanner.ts

```diff
@@ -354,26 +354,45 @@ export class DirectoryScanner implements IDirectoryScanner {
 		onError?: (error: Error) => void,
 		onBlocksIndexed?: (indexedCount: number) => void,
 	): Promise<void> {
-		if (batchBlocks.length === 0) return
+		if (batchBlocks.length === 0) {
+			console.debug("[DirectoryScanner] Skipping empty batch processing")
```

Comment:
Wait... why is the timestamp PR modifying the CBI scanner :-)

SU-PER-NO-VA!!!!! <shakes-fist />

Reply:
Yeah, thats me and git not getting along, not the model :).  I've corrected it.